### DISCUSSION TITLE
사용자 옆으로 텔레포트 하는 기능 어떨까요?

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#28] [제보하기]
poke로 부르는거 말고, 내가 상대방 옆으로 텔레포트 하는 기능은 어떤가요?
어디 있는지 찾으러 가는게 애매하네요. 
poke로 항상 부르는것도 애매하고, poke로 불러도 대답 없으면 (요놈 잡았다.) 할려고

**Comment by ACT-TJ** [comment]
의견감사합니다. 
유저리스트에서 유저를 클릭하시면 찾지않고 부를 수 있도록 되어있습니다.
메뉴얼에 빠져있는 지도 확인해 보도록 할게요!

거꾸로 내가 찾아가는 컨셉도 멋진 아이디어 같습니다.
voc 는 한번에 모아 Alpha 기간 내 피드백 드릴 수 있는 부분은 릴리즈 검토 하도록 하겠습니다^^

**Comment by shlee303** [comment]
@DolDolDol 제안 주신 덕분에 해당 기능이 배포됐습니다! :) 감사합니다~ 편하게 사용해봐주세요!
